packages/rocketchat-api/server/v1/users.js

```diff
@@ -208,7 +208,10 @@ RocketChat.API.v1.addRoute('users.update', { authRequired: true }, {
 
 			const userData = _.extend({ _id: this.bodyParams.userId }, this.bodyParams.data);
 
+			///////////// Please review this line. It will fail if we just want to enable user using REST API
+			////////////  {"userId": "ID", "data": { "active": true }}
 			RocketChat.saveUser(this.userId, userData);
```

Comment:
It doesn't fail with error. But you have provide either of this "username", "email", "name" or "password" if you just want to activate user ("active":"true").